iron-form.html

```diff
@@ -135,27 +158,25 @@
       */
 
       attached: function() {
-        this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
```

Comment:
Yeah, i don't think they do. I know we do magic so that you can use `<slot>` in v0, but that's basically a way to rename it to `content`, and `content` never had a `slotchange` equivalent